docs/guide/search/search.md

```diff
@@ -2,10 +2,39 @@
 
 eZ Platform exposes a very powerful [Search API](../../api/public_php_api_search.md), allowing both full-text search and querying the content Repository using several built-in Search Criteria and Sort Clauses. These are supported across different search engines, allowing you to plug in another search engine without changing your code.
 
-Currently two search engines exist in their own eZ Platform Bundles:
+Currently, the following search engines exist in their own eZ Platform Bundles:
 
 1.  [Legacy search engine](search_engines.md#legacy-search-engine-bundle), a database-powered search engine for basic needs.
-1.  [Solr](solr.md), an integration providing better overall performance, much better scalability and support for more advanced search capabilities **(recommended)**
+1.  [Solr](solr.md), an integration providing better overall performance, much better scalability and support for more advanced search capabilities.
+1.  [Elasticsearch](elastic.md), available for eZ Enterprise customers, a document-oriented engine providing even better performance and scalability.
+
+### Feature comparison
+
+| **Feature** | **Elasticsearch** | **Apache Solr** | **Legacy Search Engine** _**(SQL)**_ |
+| --- | --- | --- | --- |
+| Filtering | Yes | Yes | Yes, but limited\* |
+| Query _(filter with scoring)_ | Yes | Yes | Only filters, no scoring |
+| Fulltext | Yes | Yes, incl. advanced features | Partly\*\* |
+| Faceting | Partly (deprecated) | [Partly as of v1.4](https://doc.ezplatform.com/en/latest/api/public_php_api_browsing/#performing-a-faceted-search) | No |
```

Comment:
```suggestion
| Faceting (deprecated)  | Partial support | Partial support | No |
```
The oldest supported version uses the solr bundle > v1.4, so this is irrelevant.
Also remember to remove the "deprecated" note for faceting when merging to other versions (it is only deprecated in v3.2).